cpp/include/Ice/Communicator.h

```diff
@@ -199,129 +187,121 @@ namespace Ice
         /// @see Connection::setAdapter
         void setDefaultObjectAdapter(ObjectAdapterPtr adapter);
 
-        /// Get the implicit context associated with this communicator.
+        /// Gets the implicit context associated with this communicator.
         /// @return The implicit context associated with this communicator; returns null when the property
-        /// Ice.ImplicitContext is not set or is set to None.
+        /// `Ice.ImplicitContext` is not set or is set to None.
         [[nodiscard]] ImplicitContextPtr getImplicitContext() const noexcept;
 
-        /// Get the properties for this communicator.
+        /// Gets the properties for this communicator.
         /// @return This communicator's properties.
-        /// @see Properties
         [[nodiscard]] PropertiesPtr getProperties() const noexcept;
 
-        /// Get the logger for this communicator.
+        /// Gets the logger for this communicator.
         /// @return This communicator's logger.
-        /// @see Logger
         [[nodiscard]] LoggerPtr getLogger() const noexcept;
 
-        /// Get the observer resolver object for this communicator.
-        /// @return This communicator's observer resolver object.
+        /// Gets the observer object for this communicator.
+        /// @return This communicator's observer object.
         [[nodiscard]] Instrumentation::CommunicatorObserverPtr getObserver() const noexcept;
 
-        /// Get the default router for this communicator.
+        /// Gets the default router for this communicator.
         /// @return The default router for this communicator.
-        /// @throws CommunicatorDestroyedException if the communicator has been destroyed.
+        /// @throws CommunicatorDestroyedException Thrown when the communicator is destroyed.
         /// @see #setDefaultRouter
-        /// @see Router
         [[nodiscard]] std::optional<RouterPrx> getDefaultRouter() const;
 
-        /// Set a default router for this communicator. All newly created proxies will use this default router. To
-        /// disable the default router, null can be used. Note that this operation has no effect on existing proxies.
-        /// You can also set a router for an individual proxy by calling the operation <code>ice_router</code> on the
-        /// proxy.
-        /// @param rtr The default router to use for this communicator.
+        /// Sets a default router for this communicator. All newly created proxies will use this default router. This
+        /// function has no effect on existing proxies.
+        /// @param rtr The default router to use for this communicator. May be nullopt.
```

Comment:
Maybe mention nullopt purposes it to clean existing router.

Reply:
Done